pkg/build/api/types.go

```diff
@@ -42,8 +49,8 @@ type Build struct {
 // BuildSpec encapsulates all the inputs necessary to represent a build.
 type BuildSpec struct {
 	// ServiceAccount is the name of the ServiceAccount to use to run the pod
-	// created by this build.
-	// The pod will be allowed to use secrets referenced by the ServiceAccount
+	// created by this build. The pod will be allowed to use secrets referenced
```

Comment:
@rhcarvalho s/69/79. I used that as an example. Changing indentation to 79 width in this PR is unfortunate because it really makes it harder to review the changes and to me change that big meets the bar of doing separate PR, presumably having discussion under it whether that change is appropriate or not. 

Now, "exaggeration"...  

>I would have cosmetic changes in separate commits. But our single-commit approach and you comment about "unrelated changes" push us against making trivial improvements.

what I was referring in my comment was what you suggested... "I would have cosmetic changes in separate commits"... by doing that I can't just click "Files changes" tab and do review work, but I will have to go each commit and filter out those I'm not interested in....